.github/workflows/go-build.yml

```diff
@@ -0,0 +1,61 @@
+# This workflow will build a golang project
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-go
+
+name: Go Build
+
+on:
+  push:
+    branches: [ "main" ]
```

Comment:
I prefer the itemised list format, makes diffs easier to read

Reply:
Changed